ansible/pbTestScripts/updateBoxes.sh

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
+
+FORCE=""
+REMOVE=false
+
+Usage(){
+  echo "
+Usage: ./updateBoxes.sh [options]
+
+Bash script to update vagrant boxes and remove old versions.
```

Comment:
I'd add something to this sentence making it clear what happens without options. e.g. `Running with no parameters will query the system for outdated boxes and update them` or similar.